metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -2348,10 +2348,12 @@ class MetalsLanguageServer(
         case None =>
           Future.successful(BuildChange.None)
       }
-      _ = {
-        tables.dismissedNotifications.ImportChanges.reset()
-        treeView.init()
-      }
+      _ = maybeSession.map(session =>
+        if (!session.main.isSbt)
+          tables.dismissedNotifications.ImportChanges.reset()
```

Comment:
Actually this seems to have been added for https://github.com/scalameta/metals/commit/6b748f8db10e6fcdfc6c1f63b9ba190cccbb589a
and this might have been done to fix a test, so we can probably just remove it altogether and just check if the test passes still. If the test doesn't pass we might need to investigate why:
`~slow/testOnly tests.sbt.SbtServerSuite -- *reload*`

should run it.